components/dashboard/src/settings/Preferences.tsx

```diff
@@ -55,52 +51,6 @@ export default function Preferences() {
     };
     migrationIDESettings();
 
-    const updateUserIDEInfo = async (selectedIde: string, useLatestVersion: boolean) => {
```

Comment:
I've extracted the logic to present & save the IDE preference into a dedicated component `<SelectIDE>`

Reply:
`updateUserIDEInfo` was also moved to its own file, to allow reusability